### PR TITLE
User can compare stats with another user.

### PR DESCRIPTION
**Pull request by peterhermansen**
user can compare

### CODE REVIEW

**Review by thebearingedge — APPROVED** [review]
Will it be possible to remove a player from the comparison?

**Comment by peterhermansen** [comment]
User can now remove players that were previously compared.

File: main.js [hunk 1]
```diff
@@ -209,12 +211,19 @@ var $search = document.querySelector('#search-top')
...
+
+$siteLogo.addEventListener('click', writeLeaderBoard)
 
 function removeElement(element) {
   element.className += ' hidden'
   element.innerHTML = ''
 }
 
+function showElement(element) {
+  element.className -= ' hidden'
```
Comment (thebearingedge) [inline main.js]: Does this actually work?

**Review by thebearingedge — CHANGES_REQUESTED** [review]

File: main.js (hunk 1 above)
Comment (thebearingedge) [reply main.js]: It might give you the result you want, but I don't think it's doing what you expected.

**Comment by thebearingedge** [comment]
Please reopen when ready.

**Review by thebearingedge — APPROVED** [review]

**Pull request merged**